src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -3017,11 +3017,879 @@ partial class C
+                {
+                    public partial int P
+                    {
+                        get => 1;
+                        [param: A(2), B(2)]
+                        set { }
+                    }
+
+                    public partial int P
+                    {
+                        get;
+                        [param: A(1), B(1)]
+                        set;
+                    }
+                }
+                """;
+
+            var comp = CreateCompilation(source);
+            comp.VerifyEmitDiagnostics();
+
+            var accessor = comp.GetMember<MethodSymbol>("C.set_P");
+            AssertEx.Equal([], accessor.GetAttributes().SelectAsArray(a => a.ToString()));
+            AssertEx.Equal([], accessor.PartialImplementationPart.GetAttributes().SelectAsArray(a => a.ToString()));
+            AssertEx.Equal(["A(2)", "B(2)", "A(1)", "B(1)"], accessor.Parameters.Single().GetAttributes().SelectAsArray(a => a.ToString()));
+            AssertEx.Equal(["A(2)", "B(2)", "A(1)", "B(1)"], accessor.PartialImplementationPart!.Parameters.Single().GetAttributes().SelectAsArray(a => a.ToString()));
+        }
+
+        [Fact]
+        public void Attributes_Indexer()
+        {
+            var source = """
+                #pragma warning disable CS9113 // Primary constructor parameter is unread
+
+                using System;
+                
+                [AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
+                class A(int i) : Attribute { }
+                [AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
+                class B(int i) : Attribute { }
+
+                partial class C
+                {
+                    [A(2), B(2)]
+                    public partial int this[int i] { get => 1; set { } }
+
+                    [A(1), B(1)]
+                    public partial int this[int i] { get; set; }
+                }
+                """;
+
+            var comp = CreateCompilation(source);
+            comp.VerifyEmitDiagnostics();
+
+            var indexer = (SourcePropertySymbol)comp.GetMember<NamedTypeSymbol>("C").Indexers.Single();
+            AssertEx.Equal(["A(1)", "B(1)", "A(2)", "B(2)"], indexer.GetAttributes().SelectAsArray(a => a.ToString()));
+            AssertEx.Equal(["A(1)", "B(1)", "A(2)", "B(2)"], indexer.PartialImplementationPart!.GetAttributes().SelectAsArray(a => a.ToString()));
+        }
+
+        [Fact]
+        public void Attributes_IndexerParameter()
+        {
+            // Unlike other symbol kinds, for parameters, the implementation part attributes are emitted first, then the definition part attributes.
+            // This is consistent with partial methods.
+            var source = """
+                #pragma warning disable CS9113 // Primary constructor parameter is unread
+
+                using System;
+                
+                [AttributeUsage(AttributeTargets.All, AllowMultiple = true)]
```

Comment:
nit: `AttributeUsage` seems unnecessary. Consider removing (from most tests)

Reply:
AllowMultiple is reducing "noise" errors in many of the tests